draft-irtf-nwcrg-coding-and-congestion-04.xml

```diff
@@ -294,12 +294,12 @@
         	</artwork>
 		</figure>
 	
-		<t><xref target="fig:fec-in"></xref> presents an architecture where FEC is within the transport. The repair symbols are sent within what the congestion window allows, such as in <xref target="CTCP"/>.</t>
+		<t><xref target="fig:fec-in"></xref> presents an architecture where FEC operates within the transport. The repair symbols are sent within what the congestion window allows, such as in <xref target="CTCP"/>.</t>
 	</section>
 
 	<section anchor="sec:fec-in-discu" title="Discussion">
 		<t>The advantage of this approach allows a joint optimization between the CC and the FEC. Moreover, the transmission of repair symbols does not add congestion in potentially congested networks but helps repair lost packets (such as tail losses).</t>
-		<t>For reliable transfers, including redundancy reduces goodput for large file transfers but the amount of repaired symbols can be adapted, e.g. depending on the congestion window size. There is a trade-off between the cost in capacity used to transmit source packets and the benefits brought out by transmitting repair symbols (e.g. unlocking the receive buffer if this is limiting). The coding ratio needs to be carefully designed. For small files, sending repair symbols when there is no more data to transmit or preferably sending repair symbols instead of the following packets in the send buffer could help to reduce the transfer time. This is because it could avoid a silent period between the transmission of the last packet and firing the retransmission of lost packets.</t>
+		<t>For reliable transfers, including redundancy reduces goodput for large file transfers but the amount of repaired symbols can be adapted, e.g. depending on the congestion window size. There is a trade-off between the cost in capacity used to transmit source packets and the benefits brought out by transmitting repair symbols (e.g. unlocking the receive buffer if this is limiting). The coding ratio needs to be carefully designed. For small files, sending repair symbols when there is no more data to transmit could help to reduce the transfer time. This is because it could avoid a silent period between the transmission of the last packet and firing the retransmission of lost packets.</t>
```

Comment:
```suggestion
		<t>For reliable transfers, including redundancy reduces goodput for large file transfers but the amount of repaired symbols can be adapted, e.g. depending on the congestion window size. There is a trade-off between the cost in capacity used to transmit source packets and the benefits brought out by transmitting repair symbols (e.g. unlocking the receive buffer if this is limiting). The coding ratio needs to be carefully designed. For small files, sending repair symbols when there is no more data to transmit could help to reduce the transfer time. In general, sending repair symbols could avoid a silent period between the transmission of the last packet in the sending buffer and firing the retransmission of lost packets or the transmission of new packets.</t>
```

Reply:
What do you think ?

Reply:
I like that it's more precise, but now it's hard to read with all the "and"s and "or"s in one sentence. Without changing your content, here's a suggestion to make it a bit easier to read:

```suggestion
		<t>For reliable transfers, including redundancy reduces goodput for large file transfers but the amount of repaired symbols can be adapted, e.g. depending on the congestion window size. There is a trade-off between the cost in capacity used to transmit source packets and the benefits brought out by transmitting repair symbols (e.g. unlocking the receive buffer if this is limiting). The coding ratio needs to be carefully designed. For small files, sending repair symbols when there is no more data to transmit could help to reduce the transfer time. In general, sending repair symbols could avoid a silent period between the transmission of the last packet in the send buffer and 1) firing the retransmission of lost packets, or 2) the transmission of new packets.</t>
```

Reply:
Perfect!